reactor-core/src/test/java/reactor/core/publisher/FluxConcatWithTest.java

```diff
@@ -99,4 +99,15 @@ public void dontBreakFluxArrayConcatMap() {
 		.assertNoError()
 		.assertComplete();
 	}
+
+	@Test
+    public void endWith() {
```

Comment:
nitpick: rename to `concatWithValues` and prefer using `StepVerifier` (rest of the test suite is old and didn't switch to this pattern, but newer tests should prefer `StepVerifier`)